### ISSUE TITLE
Spacers for reserved rune tickers

### ISSUE THREAD

**Issue by arik-so** [ordinals/ord#3691]
When a rune ticker gets assigned from the reserved space, spacers are still taken into consideration.

Given that it's pretty difficult to know the assigned ticker a priori, the spacer value is likely nonsensical. Should that remain the intended behavior?

**Comment by cryptoni9n** [comment]
> When a rune ticker gets assigned from the reserved space, spacers are still take into consideration.
>
I'm not sure I understand what you mean.  Can you provide an example?  

> Given that it's pretty difficult to know the assigned ticker a priori, the spacer value is likely nonsensical. Should that remain the intended behavior?

Are you just asking about the spacer field in the yaml?

**Comment by cryptoni9n** [comment]
Yes, I understand that much.  But what about them?  What do you mean by this?
> spacers are still take into consideration.

**Comment by casey** [comment]
@arik-so means that if you etch a reserved rune, you can still include a spacers field, which will put spacers into the reserved rune's name. Reserved rune names are automatically assigned, so it doesn't make a lot of sense to put spacers into them.

We could ignore the spacers tag, if present on a reserved rune. We could even go so far as to always include spacers in reserved rune names, in fixed locations, for readability, like every three characters:

```
AAA•AAA•AAA•AAA•AHL•HXC•ZMX•SYU•MQP
```

Every four characters probably makes more sense, although reserved rune names are 27 characters, so it isn't evenly divisible by four.

**Comment by a709090319** [comment]
TX：ec5a4d408a2c9dba5f7377ecc678d2bd9bce34424367f2aa967a32ba9b2a990d
TX: f037e1024ad04ba000a3cab7e0d50bfe462b4c6dbac39913f05027947d575891
No need to keep the sat mint rune. How is it implemented

**Comment by summraznboi** [comment]
It's nonsensical, but I would only recommend a default preferred view if spacers was not specified, if it is, I would want to follow whatever the user specified (most prefix characters are stable given the block it will be mined in).
